src/main/java/org/gridsuite/study/server/networkmodificationtree/dto/BuildStatus.java

```diff
@@ -32,6 +35,10 @@ public static BuildStatus fromApplicationStatus(NetworkModificationResult.Applic
         }
     }
 
+    public static BuildStatus returnHigherSeverityStatus(BuildStatus buildStatus1, BuildStatus buildStatus2) {
```

Comment:
Perhaps better to use an attribute severityLevel like `ApplicationStatus` and a method max
This solution does not depend of a java implementation of enum (ordinal) 

    NOT_BUILT (-1),
    BUILDING (-1),
    BUILT (0),
    BUILT_WITH_WARNING (1),
    BUILT_WITH_ERROR (2);

Reply:
Yes I'm doing it

Reply:
Done